imports/plugins/core/graphql/server/resolvers/util/applyPaginationToMongoCursor.js

```diff
@@ -13,8 +13,8 @@
 export default async function applyPaginationToMongoCursor(cursor, { first, last } = {}, totalCount) {
   if (first && last) throw new Error("Request either `first` or `last` but not both");
 
-  // Enforce a `first: 50` limit if no user-supplied limit
-  const limit = first || last || 50;
+  // Enforce a `first: 20` limit if no user-supplied limit
+  const limit = first || last || 20;
```

Comment:
Can you add `const DEFAULT_LIMIT = 20;` at the top of the file and use that const here? We'll eventually put this in ENV but that's at least a little less magic for now.